### PR TITLE
Fix undeclared global variable "rules"

### PR DESCRIPTION
**Pull request by Rui914**

### CODE REVIEW

File: digilines/internal.lua [hunk 1]
```diff
@@ -20,12 +20,9 @@ function digiline:getAnyInputRules(pos)
 
 	if spec.wire then
 		return digiline:importrules(spec.wire.rules, node)
-	end
-	if spec.effector then
+	elseif spec.effector then
 		return digiline:importrules(spec.effector.rules, node)
```
Comment (sofar) [inline digilines/internal.lua]: are ```spec.wire``` and ```spec.effector``` mutually exclusive? if not then this is wrong...

File: digilines/internal.lua [hunk 2]
```diff
@@ -35,8 +32,7 @@ function digiline:getAnyOutputRules(pos)
 
 	if spec.wire then
 		return digiline:importrules(spec.wire.rules, node)
-	end
-	if spec.receptor then
+	elseif spec.receptor then
 		return digiline:importrules(spec.receptor.rules, node)
 	end
```
Comment (sofar) [inline digilines/internal.lua]: same question here.

File: digilines/internal.lua (hunk 1 above)
Comment (ssieb) [inline digilines/internal.lua]: It doesn't actually change the logic.  However, it does turn it into an "else after return", so this change should be reverted

**Comment by ssieb** [comment]
Which part of this change is related to the subject?  Please split the "cleanup" changes into a separate change.
Why do you feel it is necessary to change all the quotes?

**Comment by Rui914** [comment]
Fixed.

**Pull request merged**